### DISCUSSION TITLE
Releases changelog or description

### DISCUSSION THREAD

**Discussion by info-bit** [JustFly1984/react-google-maps-api#2654] [Q&A]
It would be great to have a very short and concise changelog of every new release. Keeping the dependencies updated in a large project without having a changelog is very painful. What do you think?

**Comment by jacklimwenjie** [comment]
I second this, especially the major version bump - I have no idea if anything is broken when upgrading from version 1 to version 2. (BTW does anyone know what are the breaking changes, if any?)

It would also be great if the tag on Github is being updated but sadly it isn't anymore since version 1.9.5.
